client/components/thank-you-v2/upsell/style.scss

```diff
@@ -67,7 +67,7 @@
 		gap: 16px;
 	}
 
-	a.components-button {
+	.components-button {
```

Comment:
When we pass `{ disabled: true }` to the action button, it renders as a `<button>` instead of as an `<a>`, but we still want the styling to apply.